.env

```diff
@@ -0,0 +1 @@
+JWT_SECRET="nobody tosses a dwarf!"
```

Comment:
Put your `.env` file into your `.gitignore` so that the secret won't be uploaded to github. Even though it's not as important on this project, it's a good thing to make a habit of.